src/stream/Middleware.ts

```diff
@@ -117,6 +117,15 @@ export class OrderedHistoryResumer extends MiddlewareBase {
     this.slidingWindow.subscribe(this.onMessage.bind(this));
   }
 
+  applyHistory() {
```

Comment:
We should mark this method `public` and move it alongside the other public methods